website/files/models/base.py

```diff
@@ -199,7 +199,7 @@ class StoredFileNode(StoredObject, Commentable):
     checkout = fields.AbstractForeignField('User')
 
     #Tags for a file, currently only used for osfStorage
-    tags = fields.ForeignField('Tag', list=True, index=True)
+    tags = fields.ForeignField('Tag', list=True)
```

Comment:
Have you talked with anyone about the performance implications of this? Is it otherwise handled somehow, or is filtering on tags just not going to work well?